### PR TITLE
Proposed fixes for MixBytes audit

### PR DESCRIPTION
**Pull request by vp4242**
Proposed fixes for MixBytes audit

### CODE REVIEW

File: contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol [hunk 1]
```diff
@@ -923,7 +925,6 @@ contract OracleReportSanityChecker is AccessControlEnumerable {
     error IncorrectCLBalanceIncrease(uint256 annualBalanceDiff);
     error IncorrectAppearedValidators(uint256 churnLimit);
```
Comment (TheDZhon) [inline contracts/0.8.9/sanity_checks/OracleReportSanityChecker.sol:926]:
```suggestion
    error IncorrectAppearedValidators(uint256 churnLimit, uint256 actualAppearedValidators);
```

**Review by TheDZhon — CHANGES_REQUESTED** [review]
:eyes:

**Review by TheDZhon — APPROVED** [review]
LGTM

**Review by folkyatina — APPROVED** [review]
:shipit:

**Review by avsetsin — APPROVED** [review]
lgtm

**Pull request merged**